website/docs/docusaurus.config.js.md

```diff
@@ -3,9 +3,245 @@ id: docusaurus.config.js
 title: docusaurus.config.js
 ---
 
-TODO: Explain the various `docusaurus.config.js` options.
+<!--
+  Goal: To serve as a manual for all configurations in docusaurus.config.js
+  Should keep the titles to themselves for cleaner link
+-->
 
-#### References
+## Overview
 
-- https://docusaurus.io/docs/en/site-config
-- https://v1.vuepress.vuejs.org/config/
+`docusaurus.config.js` contains configurations for your site and is placed in the root directory of your site.
+
+```ts
+interface DocusaurusConfig {
+  baseUrl: string;
+  favicon: string;
```

Comment:
one last nit. Lets avoid putting this (?) typescript can be intimidating

Reply:
Do you refer to the whole config object or the question mark on the optional field?

Reply:
the whole config object.

Reply:
For example ys just changed the interface a little bit too